### PR TITLE
use preload exclusions from the backend

### LINKED ISSUE #5810: 403 URLs arenot removed from caching table
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version => 3.12.6.1
- Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
Using Ultimate Member plugin, will add URL to cache table like `https://new.rocketlabsqa.ovh/void(0);`, this URL isnot removed once processed 

**To Reproduce**
- Ultimate Member plugin is activated
- WP sitemap is disabled (URLs fetched from home page)
1. install and activate WPR
2. Check cache table => mentioned URL is added and once processed will fail

**Expected behavior**
403 URLs shall be removed as same as 404 URLs

**Screenshots**
Screenshot from 2023-03-13 11-55-03

**Additional context**
Screenshot from 2023-03-13 12-25-17
We shouldn't fetch this kind of URL.

**Backlog Grooming (for WP Media dev team use only)**
- [x] Reproduce the problem
- [x] Identify the root cause
- [x] Scope a solution
- [x] Estimate the effort

### PR DESCRIPTION
**Pull request by mostafa-hisham**
## Description

Use the newly added preload_excluison in the backend on the plugin side

Fixes #5810 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Enhancement (non-breaking change which improves an existing functionality)
- [ ] This change requires a documentation update

## Is the solution different from the one proposed during the grooming?

No
## How Has This Been Tested?

- [ ] Unit and integration tests

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/Integration/inc/Engine/Optimization/DynamicLists/Subscriber/addPreloadExcluisons.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+namespace WP_Rocket\Tests\Integration\inc\Engine\Optimization\DynamicLists\Subscriber;
+
+use WP_Rocket\Tests\Integration\TestCase;
+
+/**
+ * @covers \WP_Rocket\Engine\Optimization\DynamicLists\Subscriber::add_preload_exclusions()
+ *
+ * @group  DynamicLists
+ */
+class Test_AddPreloadExcluisons extends TestCase {
```
Comment (remyperona) [inline tests/Integration/inc/Engine/Optimization/DynamicLists/Subscriber/addPreloadExcluisons.php]:
```suggestion
class Test_AddPreloadExclusions extends TestCase {
```

+ same typo in the filename

File: tests/Fixtures/inc/Engine/Optimization/DynamicLists/Subscriber/addPreloadExcluisons.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?php
```
Comment (remyperona) [inline tests/Fixtures/inc/Engine/Optimization/DynamicLists/Subscriber/addPreloadExcluisons.php]: fix typo in the filename

**Review by remyperona — CHANGES_REQUESTED** [review]

**Review by remyperona — APPROVED** [review]

**Review by vmanthos — APPROVED** [review]

**Comment by vmanthos** [comment]
This is working as expected. :+1: 

PS: The exclusion I've added to the staging SaaS back end is `void\(.*;`. 
This should be added to the production SaaS once the PR that adds the preload exclusions is merged there.

**Comment by vmanthos** [comment]
@mostafa-hisham Can you push to the SaaS production the related branch? :pray:

**Comment by mostafa-hisham** [comment]
Done

**Pull request merged**